portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase

```diff
@@ -253,8 +253,40 @@ definition {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132278 PredefinedValueIsResetWhenSelectedOptionIsDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Multiple Selection");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Option 1",
+			optionValue = "1");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Option 2",
+			optionValue = "2");
```

Comment:
Hey @piresdm, can we rename this macro to `addOption` only?

Apparently, it was created firstly to attend Single Selection field needs, after all its current name is `addSingleSelectionOption`, although this same macro is being invoked on tests related to the fields **Single Selection**, **Multiple Selection** and **Select From List**. Therefore, in order to better reflect the actual use of this macro would be important we rename it. :slightly_smiling_face:

Reply:
@natocesarrego like we talked before, this macro was used in many other PR`s that were already merged. So, I did not change it...